react/src/components/FoodFeed.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import Card from './Card';
+
+class FoodFeed extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      foods: []
+    }
+  }
+
+  retrieveFoods() {
+    $.ajax({
```

Comment:
Fetch is fine to use here since you're already in ES6! AJAX is only used in plain JS files.